### PR TITLE
prepare for code review

### PR DESCRIPTION
**Pull request by Gr33ndev**

### CODE REVIEW

**Review by WuestFlorian — APPROVED** [review]

File: Assets/Scripts/Barriers/BarrierData.cs [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-using System.Collections.Generic;
-using UnityEngine;
-
-/*
- * This scripts defines the barrier data that needs to be transfered from the game manager to a barrier object.
- */
+/// <summary>
+///     This class defines all needed data for a
+///     <c>
+///         <Barrier/ c>.
```
Comment (marlau97) [inline Assets/Scripts/Barriers/BarrierData.cs:4]:
```suggestion
///     This class defines all needed data for a <c>Barrier</c>.
```

File: Assets/Scripts/GameManager/DTOs/PlayerTaskStatisticDTOArray.cs [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
 public class PlayerTaskStatisticDTOArray
 {
+    #region Attributes
+
     public PlayerTaskStatisticDTO[] playerTaskStatisticDTOs;
 
+    #endregion
+
+    /// <summary>
+    ///     This function converts a json string to a array of<c>PlayerTaskStatisticDTO</c> objects.
```
Comment (marlau97) [inline Assets/Scripts/GameManager/DTOs/PlayerTaskStatisticDTOArray.cs:17]:
```suggestion
    ///     This function converts a json string to a array of <c>PlayerTaskStatisticDTO</c> objects.
```

File: Assets/Scripts/HUD/CharacterSelection.cs [hunk 3]
```diff
@@ -1,13 +1,20 @@
 using UnityEngine;
 using UnityEngine.UI;
 
+/// <summary>
+/// 
+/// </summary>
```
Comment (marlau97) [inline Assets/Scripts/HUD/CharacterSelection.cs:6]:
```suggestion
```
remove unecessary empty comment

File: Assets/Scripts/Interactable/Interaction.cs [hunk 4]
```diff
@@ -1,16 +1,20 @@
 using UnityEngine;
 
+/// <summary>
+/// 
+/// </summary>
```
Comment (marlau97) [inline Assets/Scripts/Interactable/Interaction.cs:5]:
```suggestion
```
remove unecessary empty comment

File: Assets/Scripts/Interactable/Item.cs [hunk 5]
```diff
@@ -1,20 +1,24 @@
...
+/// <summary>
+///     This class manages the items that player can pick up.
+/// </summary>
 [RequireComponent(typeof(BoxCollider2D))]
-
 public class Item : MonoBehaviour
 {
     public GameObject gameObjectToDestroy;
+
     // Update is called once per frame
```
Comment (marlau97) [inline Assets/Scripts/Interactable/Item.cs:11]:
```suggestion
```
remove outdated comment

**Review by marlau97 — CHANGES_REQUESTED** [review]

**Review by marlau97 — APPROVED** [review]

**Pull request merged**